core/domain/topic_domain_test.py

```diff
@@ -425,59 +370,75 @@ def test_add_additional_story(self):
         self.assertEqual(
             additional_story_ids,
             ['story_id', 'story_id_1', 'story_id_2'])
-        with self.assertRaisesRegexp(
+        with self.assertRaisesRegexp(  # type: ignore[no-untyped-call]
             Exception, 'The story_id story_id_2 is already present in the '
             'additional story references list of the topic.'):
             self.topic.add_additional_story('story_id_2')
 
-    def _assert_validation_error(self, expected_error_substring):
+    def _assert_validation_error(  # type: ignore[override]
```

Comment:
Why is the override needed?

Reply:
Before adding this, I was getting these errors.
![d_7](https://user-images.githubusercontent.com/89520981/148721778-6c68ba4d-f755-4fe7-9836-7130cecc2192.png)
In this test file we have two methods with the same name `_assert_validation_error`, and I think for them those errors were raised. So, should I keep them?

Reply:
Yep, just add an explanatory comment for this.

Reply:
In this file `core/domain/topic_domain.py`, we are using two methods with the same name `_assert_validation_error`, and for that, Mypy checks are giving the above-mentioned errors. To silence them I have used those comments.

Reply:
Sure, but please add this info into the code.

Reply:
I have added a few comments there. Please take a look and let me know if any changes needed.

Reply:
ditto as below

Reply:
I have updated the comment. Please have a look